CHANGES.txt

```diff
@@ -50,9 +50,12 @@
 
```

Comment:
I think this should be:

> Drops support for Python <3.5.

We still support Python 3.5, though we may drop 3.5 soon also.